components/ActionSheet/src/private/MDCActionSheetHeaderView.m

```diff
@@ -199,4 +206,14 @@ - (void)mdc_setAdjustsFontForContentSizeCategory:(BOOL)adjusts {
   [self updateFonts];
 }
 
+- (void)setTitleTextColor:(UIColor *)titleTextColor {
+  _titleTextColor = titleTextColor;
+  _titleLabel.textColor = titleTextColor;
+}
+
+- (void)setMessageTextColor:(UIColor *)messageTextColor {
+  _messageTextColor = messageTextColor;
+  _messageLabel.textColor = messageTextColor;
```

Comment:
This logic doesn't match the logic of styleTitleAndMessage. Consider calling that method in each of these setters.

Reply:
In what way do you feel this doesn't match the logic? If the text color is set then that will be the label's text color. `styleTitleAndMessage` should be called if the `title` or `message` value changes. For example:

`self.message = @"Hello"`
`self.message = nil;`

That would mean that the title is now the only label and needs to be 0.6 alpha. But, the text color is the same if there is text or not. Does that make sense, I feel I didn't explain it well if there is confusion let's talk offline.

Reply:
Ah I misunderstood the fact that only the alpha is being chosen conditionally for the title label. Re-reading the code again I see now that we're 1-1 mapping these APIs (I'd thought we were doing conditional color mapping based on the text value).